README.md

```diff
@@ -90,6 +90,12 @@ One of our goals is to make the transition from ARM Templates to Bicep as easy a
 
 Note that while we want to make it easy to transition to Bicep, we will continue to support and enhance the underlying ARM Template JSON language. As mentioned in [What is Bicep?](#what-is-bicep), ARM Template JSON remains the wire format that will be sent to Azure to carry out a deployment.
 
+**What unique benefits do you get with Bicep?**
+a) Day 1 resource provider support. Any Azure resource whether in public preview or GA can be provisioned using Bicep b) No state or state files to manage. All state is stored in Azure, so makes it easy to collaborate and make changes to resources confidently. c) Tooling is the cornerstone to any great experience and our VSCode extension for Bicep makes it extremely easy to author and get started. d) Supported by Microsoft support and 100% free to use.
```

Comment:
This should be a markdown-formatted bullet or numbered list

Reply:
And I'd change "day 1" to "day 0".